packages/web/src/components/molecules/Navigation/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Menu } from 'antd'
+import { useState } from 'react'
+import { useCallback } from 'react'
+import { ClickParam } from 'antd/lib/menu'
+import { useRouter } from 'next/router'
+import styled from 'styled-components'
+
+const navs = [
+  {
+    key: 'properties',
+    label: 'Propertis',
+    pathname: '/'
+  },
+  {
+    key: 'create',
+    label: 'Create',
+    pathname: '/auth'
+  }
+]
+const NavMenu = styled(Menu)`
+  background: transparent;
+`
+
+const toPathname = (_key: string) => navs.find(({ key }) => key === _key)?.pathname
+const toKey = (_pathname: string) => navs.find(({ pathname }) => pathname === _pathname)?.key
+
+export const Navigation = () => {
+  const router = useRouter()
+  const [current, setCurrent] = useState(toKey(router?.pathname) || navs[0].key)
+  const handleClick = useCallback(
+    (e: ClickParam) => {
+      const { key } = e
+      const pathname = toPathname(key)
+      setCurrent(key)
+      pathname && router.push(pathname)
+    },
+    [setCurrent, router]
+  )
+
+  return (
+    <NavMenu onClick={handleClick} selectedKeys={[current]} mode="horizontal">
+      {navs.map(nav => (
+        <NavMenu.Item key={nav.key}>{nav.label}</NavMenu.Item>
+      ))}
+    </NavMenu>
+  )
+}
```

Comment:
In Next.js, except for situations where the application actively transitions, you should use basically Link Component. If you don't, no prefetch will be done and the performance of the transition will be extremely slow. 

Therefore, I thought you should use the nav manu item the Link Component of next js.

Reply:
Okay, I tried to use the Link component, but the Menu component was broken🤔